framework/src/outputs/CSV.C

```diff
@@ -101,7 +101,8 @@ CSV::output(const ExecFlagType & type)
     _all_data_table.printCSV(filename(), 1, _align);
```

Comment:
We really need to get in the habit of using MOOSE asserts as a team. This line here is currently only called on processor zero but the actual function being called is buried three classes down. We're just asking for this bug to pop up again.